src/models/MySpace.ts

```diff
@@ -98,7 +98,6 @@ export const MySpaceModel = {
       throw e
     }
   },
-
   async deleteWidget(
```

Comment:
Removing the space was a mistake, but beneficial so I can point out: Since `WeatherWidget` extends `Widget`, we can actually use `deleteWidget` and don't need any new mutation for removing.